variables.tf

```diff
@@ -22,6 +22,12 @@ variable "create" {
   description = "Set to false to prevent the module from creating any resources"
 }
 
+variable "permission_boundary_arn" {
+  type        = string
+  default     = null
```

Comment:
Same question as other changes

Should default be null or empty string?

Reply:
Null, here is why:
Null a value that represents absence or omission. If you set an argument of a resource to null, Terraform behaves as though you had completely omitted it — it will use the argument's default value if it has one, or raise an error if the argument is mandatory. null is most useful in conditional expressions, so you can dynamically omit an argument if a condition isn't met.
https://developer.hashicorp.com/terraform/language/expressions/types#null